mesh/mesh.go

```diff
@@ -474,6 +474,11 @@ func (m *Mesh) AccumulateRewards(rewardLayer types.LayerID, params Config) {
 			)
 			continue
 		}
+		if bl.ATXID == *types.EmptyAtxId {
+			m.With().Info("skipping reward distribution for block with no ATX",
+				log.LayerId(uint64(bl.LayerIndex)), log.BlockId(bl.Id().String()))
+			continue
+		}
```

Comment:
If there isn't, there should be a big TODO here, regarding modifications after Genesis flow is agreed upon.
We should not accept empty ATX blocks, so they should never get here.

Reply:
i  think you should open a task or  add a description of this in the "genesis flow" task

Reply:
I agree with @almogdepaz. TODOs in the code are easy to forget about, we should (additionally) document this in the issue. I'll add the TODO and document it in the issue and then link to the issue here.

Reply:
We said that we will change the issue (#1606) to handle the loose ends from this PR

Reply:
I think it's better to close that issue (when this PR is merged) and create a new one, specifically for the genesis flow.

Reply:
I've updated issue https://github.com/spacemeshos/go-spacemesh/issues/897

Reply:
I don't think it's necessary to add `TODO` comments to the code here. When we write the genesis flow, we'll look for usages of the word "genesis" in the whole code and update what's needed.